app/importers/data_transformers/csv_transformer.rb

```diff
@@ -4,8 +4,12 @@ class CsvTransformer
 
   attr_accessor :pre_cleanup_csv_size
 
+  def initialize(headers=true)
```

Comment:
For optional arguments, accepting an `options = {}` hash helps make a bit more explicit what's happening.  The call site then will read like `CsvTransformer.new(headers: whatever)` instead of the more cryptic `CsvTransformer.new(whatever)`.  And this particularly helps keep it clear what's happening when you add or remove optional arguments over time.